From 6795406a2f783bceef3f31ff19a897e2797c9d13 Mon Sep 17 00:00:00 2001
From: root <root@ntbprimary.amd.com>
Date: Tue, 23 Jul 2024 12:21:58 +0000
Subject: [PATCH 7/9] ntb_perf: Add prints to display physical and virtual
 address

Add additional prints to display virtual and physical address
of the source and destination buffers to help verify it using
HDT.

Signed-off-by: root <root@ntbprimary.amd.com>
---
 drivers/ntb/test/ntb_perf.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/drivers/ntb/test/ntb_perf.c b/drivers/ntb/test/ntb_perf.c
index 1d0b95661b0b..94b86f591c7b 100644
--- a/drivers/ntb/test/ntb_perf.c
+++ b/drivers/ntb/test/ntb_perf.c
@@ -639,6 +639,7 @@ static int perf_setup_inbuf(struct perf_peer *peer)
 		goto err_free_inbuf;
 	}
 
+	dev_err(&perf->ntb->dev, "xlat virt_addr %llx xlat dma_addr %llx ", peer->inbuf, peer->inbuf_xlat);
 	/*
 	 * We submit inbuf xlat transmission cmd for execution here to follow
 	 * the code architecture, even though this method is called from service
@@ -950,7 +951,11 @@ static int perf_init_test(struct perf_thread *pthr)
 				pthr->tidx,
 				&peer->out_phys_addr,
 				&pthr->dma_dst_addr[tidx]);
+		dev_dbg(pthr->dma_chan[tidx]->device->dev, "%d: src virt_addr %llx src dma_addr %llx phys_from_dma %llx",pthr->tidx, pthr->src[tidx], pthr->dma_src_addr[tidx], pthr->dma_src_addr[tidx]);
+		dev_dbg(pthr->dma_chan[tidx]->device->dev, "%d: dst virt_addr %llx dst phys_addr %llx dst dma_addr %llx phys_from_dma %llx",pthr->tidx, peer->outbuf, peer->out_phys_addr, pthr->dma_dst_addr[tidx], pthr->dma_dst_addr[tidx]);
+
 	}
+
 	atomic_set(&pthr->dma_sync, 0);
 	return 0;
 
-- 
2.39.2

